### PR TITLE
DOC: update oscillator docs

### PR DESCRIPTION
**Pull request by haydenrob**

### CODE REVIEW

File: demos/refellipsDemo_UserDefinedOscillator.ipynb [hunk 1]
```diff
@@ -419,7 +419,7 @@
    },
    "source": [
     "## Plotting $n, k$ as a function of wavlength\n",
-    "We can now calculate the refractive indices (`n`) and extinction coefficients (`k`) of our material from `e` and plot as a function of wavelength. You can also request this from the `Gauss` object by using the `complex` method, `gauss.complex(wavelength_range)`."
+    "We can now calculate the refractive indices (`n`) and extinction coefficients (`k`) of our material from `e` and plot as a function of wavelength."
```
Comment (andyfaff) [inline demos/refellipsDemo_UserDefinedOscillator.ipynb:422]: Why did you remove 

> You can also request this from the `Gauss` object by using the `complex` method, `gauss.complex(wavelength_range)`?

**Review by andyfaff — COMMENTED** [review]
Can you execute `getting_started` in a Jupyter notebook, not in Pycharm? It seems like Pycharm adds a whole lot of cruft to the file.

File: demos/refellipsDemo_UserDefinedOscillator.ipynb (hunk 1 above)
Comment (haydenrob) [reply demos/refellipsDemo_UserDefinedOscillator.ipynb]: I didn't think we should include it because we didn't define it explicitly here, but I've just added it back in

Comment (andyfaff) [reply demos/refellipsDemo_UserDefinedOscillator.ipynb]: It's inherited from ScattererSE.
